sdk/typescript/src/utils/api-endpoints.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export enum Network {
+  local = 0,
```

Comment:
I’d avoid numeric enums and use string values.  I’d also all caps the enum values just for consistency.